README.md

```diff
@@ -72,6 +72,7 @@ Please submit any feature enhancements, bug fixes, or ideas via pull requests or
       --ssh-common-args='-o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null' \
       --user=vagrant \
       -e "cert_names=DNS:localhost" \
+      -e "node_count=1" \
```

Comment:
Is this necessary?

Reply:
Yes, this is required for
https://github.com/dominodatalab/ranchhand/blob/a34ec5e3f3769c8787c01fdcd3bbc8f1afdb5295/ansible/roles/rancher/tasks/rancher.yml#L12